tests/test_other.py

```diff
@@ -10004,6 +10004,14 @@ def test_old_makeDynCall_syntax(self):
     err = self.run_process([EMCC, test_file('test_old_dyncall_format.c'), '--js-library', test_file('library_test_old_dyncall_format.js')], stderr=PIPE).stderr
     self.assertContained('syntax for makeDynCall has changed', err)
 
+  # Tests that dynCalls are produced in Closure-safe way in DYNCALLS mode when no actual dynCalls are used
+  @parameterized({
+    'plain': [[]],
+    'asyncify': [['-sASYNCIFY']],
+    'asyncify-bigint': [['-sASYNCIFY', '-sWASM_BIGINT']]})
```

Comment:
Huh, I'm surprised this works... `-` is not valid in a python name (these become part of the name of the test), but it does not error...

Still, I think we should use `_` here since I think it won't be possible to run the test with `./tests/runner.py other.test_NAME`?

```suggestion
    'asyncify_bigint': [['-sASYNCIFY', '-sWASM_BIGINT']]})
```